apt/method.go

```diff
@@ -206,28 +227,46 @@ func (m *Method) handleAcquire(ctx context.Context, msg *Message) error {
 	return nil
 }
 
+// Ported from apt's `StringToBool` function
+// https://salsa.debian.org/apt-team/apt/-/blob/a0a76c2e20c1ddefd76a4a539a9350b96d66006e/apt-pkg/contrib/strutl.cc#L824
+func stringToBool(s string) bool {
+	if i, err := strconv.Atoi(s); err == nil {
+		if i == 1 {
+			return true
+		}
+		return false
+	}
+
+	sl := strings.ToLower(s)
+	trueStrs := []string{"yes", "true", "with", "on", "enable"}
+	for _, trueStr := range trueStrs {
+		if sl == trueStr {
+			return true
+		}
+	}
+
+	return false
+}
+
 func (m *Method) handleConfigure(msg *Message) {
 	configs, ok := msg.fields["Config-Item"]
 	if !ok {
 		// Nothing to set.
 		return
 	}
 	for _, configItem := range configs {
-		if strings.Contains(configItem, "Acquire::gar::Service-Account-JSON") {
-			parts := strings.SplitN(configItem, "=", 2)
-			if len(parts) != 2 {
-				// TODO: log this?
-				return
-			}
-			m.config.serviceAccountJSON = strings.TrimSpace(parts[1])
+		parts := strings.SplitN(configItem, "=", 2)
+		if len(parts) != 2 {
+			// TODO(hopkiw): log this?
```

Comment:
if you believe we need to log something here, please go ahead and do it, instead get rid of this todo. We are touching the code, review it and apply it (it's a cheap enough change/decision).

Reply:
Good call. I've added a log line.